Helper/DeviceView.php

```diff
@@ -54,14 +54,12 @@ class DeviceView
      */
     public function __construct(Container $serviceContainer)
     {
-        if (false === $serviceContainer->isScopeActive('request')) {
+        if (!$this->request = $serviceContainer->get('request_stack')->getMasterRequest()) {
```

Comment:
This won't work in Symfony 2.3 or older as the request stack was introduced in Symfony 2.4.